red/My-List-Util/lib/My/List/Util.pm

```diff
@@ -3,13 +3,11 @@ package My::List::Util;
 use 5.006;
 use strict;
 use warnings;
+use parent qw( Exporter );
```

Comment:
I think the book mentioned that `use Exporter qw( import );` is the better practise version to use